### ISSUE TITLE
Add Haveno option to swap for incognito mode

### ISSUE THREAD

**Issue by detherminal** [cypherstack/stack_wallet#491]
It would be useful for privacy-consious people if we add [haveno](https://haveno.exchange/) swap option for incognito mode

**Comment by danrmiller** [comment]
Their website says this only exists for testnet.

**Comment by detherminal** [comment]
Yes, but for now. They will open the exchange to mainnet when they complete it.
